server/container/util/src/test/java/org/apache/james/util/RunnablesTest.java

```diff
@@ -0,0 +1,54 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.common.base.Stopwatch;
+import reactor.core.publisher.Flux;
+
+class RunnablesTest {
+
+    @Test
+    void shouldActuallyRunThings() {
+        AtomicBoolean sideEffect = new AtomicBoolean(true);
+        Runnables.runParallel(() -> sideEffect.set(false));
+        assertThat(sideEffect).isFalse();
```

Comment:
I would rather turn the boolean into true. Then it reads like 'yes, there was a side effect', which I find easier.

Reply:
ok